### DISCUSSION TITLE
Make P2(n) into reality.

### DISCUSSION THREAD

**Discussion by Mogotke** [ikemen-engine/Ikemen-GO#1303] [Feature Request]
### Is your feature request related to a problem? Please describe.

The wiki mentions that p2 is able to redirect to the n-th nearest alive opponent.
Unfortunately, it cannot do that, it only redirects to the nearest alive opponent, so the wiki is wrong.
`trigger1 = p2(1),Life <= 100`
Would not work in-game.

### Describe the solution you'd like

However, I would like to see it get that feature added into. 
As an example, enemynear can already do that, but it cannot work on helpers.
P2(n) would be a neat alternative to enemynear in that scenario.

### Search terms

p2 trigger enemynear helper redirect

**Comment by potsmugen** [comment]
It sounds a bit weird to me, personally. Like p2(2) referring to "the third player 2". I think other ways of checking those helpers could be achieved. Although their usage isn't recommended to begin with.

**Reply by Mogotke** [reply]
That is just a naming issue. P2 behaves similar to enemynear, with the added player-type helper detection on top.
Trying to find the 2nd nearest opponent, when said helpers are involved, becomes annoying imo.
Yes, I'm aware about their usage.
